Cargo.toml

```diff
@@ -32,6 +32,7 @@ derive_more = "0.99.16"
 env_logger = "0.9.0"
 hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
 hex-literal = "0.4.1"
+getrandom = { version = "0.2.10", default-features = false, features = ["js"] }
```

Comment:
Pretty sure that the feature JS should not be here, and has been introduced due to a wrong feature needed in std.

But maybe I am wrong.

Reply:
I introduced it because getrandom docs say the flag is needed for wasm32-unknown-unknown target